src/js/analyses/components/Pathoscope/PathoscopeDetail.tsx

```diff
@@ -19,20 +18,25 @@ export function PathoscopeDetail({ hit, mappedCount }: PathoscopeDetailProps) {
     const { isolates, pi } = hit;
 
     const filtered = filter(isolates, isolate => !filterIsolates || isolate.pi >= 0.03 * pi);
-    const isolateComponents = map(filtered, isolate => (
-        <PathoscopeIsolate
-            key={isolate.id}
-            {...isolate}
-            reads={Math.round(isolate.pi * mappedCount)}
-            showPathoscopeReads={showReads}
-        />
-    ));
+    const isolateRefs = useRef([]);
 
-    return (
-        <div>
-            <ScrollSync>
-                <div>{isolateComponents}</div>
-            </ScrollSync>
-        </div>
-    );
+    useEffect(() => {
+        isolateRefs.current = map(filtered, (_filter, index) => isolateRefs.current[index]);
+    }, [filtered]);
```

Comment:
Seems like it is just going to truncate `isolateRefs` down to the length of `filtered`. Would be much simpler to do via slice if that's the goal

Related, it would be a huge boon for a usability of the hook to abstract needing to manage the refs entirely out of the calling component